src/status_im2/subs/wallet/wallet_test.cljs

```diff
@@ -86,16 +86,43 @@
           :public-key               "0x04371e2d9d66b82f056bc128064"
           :removed                  false}})
 
+(def network-data
+  {:test [{:test?            true
+           :short-name       "eth"
+           :network-name     :ethereum
+           :related-chain-id 1
+           :layer            1}
+          {:test?            true
+           :short-name       "arb1"
+           :related-chain-id 42161
+           :layer            2}
+          {:test?            true
+           :short-name       "opt"
+           :related-chain-id 10
+           :layer            2}]
+   :prod [{:test?      false
+           :short-name "eth"
+           :chain-id   1
+           :layer      1}
+          {:test?      false
+           :short-name "arb1"
+           :chain-id   42161
+           :layer      2}
+          {:test?      false
+           :short-name "opt"
+           :chain-id   10
+           :layer      2}]})
+
 (h/deftest-sub :wallet/balances
   [sub-name]
   (testing "returns seq of maps containing :address and :balance"
     (swap! rf-db/app-db #(-> %
                              (assoc-in [:wallet :accounts] accounts)
                              (assoc :wallet/tokens tokens)))
-    (is (= `({:address "0x1"
-              :balance 3250}
-             {:address "0x2"
-              :balance 2100})
+    (is (= [{:address "0x1"
+             :balance 3250}
+            {:address "0x2"
+             :balance 2100}]
            (rf/sub [sub-name])))))
```

Comment:
Why did we change the list for a vector?

although they can be compared as the same, I'd prefer not changning it, since when you `conj` the behavior is different.

Reply:
Ok, I will fix only quoting.